src/lib/types.ts

```diff
@@ -0,0 +1,11 @@
+import {
+  Locale,
+  DrupalMenuLinkContent,
+} from "next-drupal"
+
+export declare type HeaderProps = {
+  locale: Locale,
+  menu?: DrupalMenuLinkContent[],
+  themes?: DrupalMenuLinkContent[],
+  langLinks?: any,
```

Comment:
With typescript declarations we should use either ';' or nothing at the end of the line instead of ','